Compiled from "FailedSanityCheckException.java"
public class org.apache.hadoop.hbase.exceptions.FailedSanityCheckException extends org.apache.hadoop.hbase.DoNotRetryIOException {
  public org.apache.hadoop.hbase.exceptions.FailedSanityCheckException();
    Code:
       0: aload_0
       1: invokespecial #1                  // Method org/apache/hadoop/hbase/DoNotRetryIOException."<init>":()V
       4: return

  public org.apache.hadoop.hbase.exceptions.FailedSanityCheckException(java.lang.String);
    Code:
       0: aload_0
       1: aload_1
       2: invokespecial #2                  // Method org/apache/hadoop/hbase/DoNotRetryIOException."<init>":(Ljava/lang/String;)V
       5: return

  public org.apache.hadoop.hbase.exceptions.FailedSanityCheckException(java.lang.String, java.lang.Throwable);
    Code:
       0: aload_0
       1: aload_1
       2: aload_2
       3: invokespecial #3                  // Method org/apache/hadoop/hbase/DoNotRetryIOException."<init>":(Ljava/lang/String;Ljava/lang/Throwable;)V
       6: return
}
